packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -780,13 +864,19 @@ contract Attestations is
       currentIndex++;
       attestations.status = AttestationStatus.Incomplete;
       // solhint-disable-next-line not-rely-on-time
-      attestations.time = uint128(now);
+      attestations.blockNumber = uint64(block.number);
       state.issuers.push(issuer);
     }
   }
 
-  function isAttestationTimeValid(uint128 attestationTime) internal view returns (bool) {
+  function isAttestationRequestBlockTimeValid(
```

Comment:
I think this function name could be more readable, maybe `isAttestationRequestExpired`?

Reply:
Named it to `isAttestationExpired` following the below conversation on what really constitutes a request